README.md

```diff
@@ -1,5 +1,11 @@
 # Yak
 
+[![GitHub Actions status](https://github.com/ros-industrial/yak/workflows/CI/badge.svg?branch=devel)](https://github.com/ros-industrial/yak/actions)
+[![GitHub issues open](https://img.shields.io/github/issues/ros-industrial/yak.svg?)](https://github.com/ros-industrial/yak/issues)
+[![license - MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![support level: community](https://img.shields.io/badge/support%20level-community-lightgray.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)
```

Comment:
#48 suggests/asks whether this should be `consortium` instead.

Perhaps @schornakj can decide later today.

Reply:
@gavanderhoorn @ipa-rwu Based on further discussion of the support levels in IRC, let's use the `consortium` badge for Yak.

Reply:
@schornakj: you should be able to push new commits to this PR.

Reply:
I updated the badge, please give a final check and merge: https://github.com/ros-industrial/yak/pull/49/commits/494e2928d9bd65862fb514bcd4f62120d1f4fa5e